tests/beaker_tests/cisco_vxlan_vtep_vni/test_vxlan_vtep_vni.rb

```diff
@@ -353,6 +387,14 @@ def test_harness_cisco_vxlan_vtep_vni(tests, id)
   # tests[id][:desc] = '2.8 Suppress ARP'
   # test_harness_cisco_vxlan_vtep_vni(tests, id)
 
+  # id = 'suppress_uuc_true'
+  # tests[id][:desc] = '2.9 Suppress Unknown Unicast'
+  # test_harness_cisco_vxlan_vtep_vni(tests, id)
+  #
+  # id = 'suppress_uuc_false'
+  # tests[id][:desc] = '2.10 Suppress Unknown Unicast'
+  # test_harness_cisco_vxlan_vtep_vni(tests, id)
```

Comment:
What's the plan for these commented out tests?

Also, just an fyi -- the minitests are responsible for testing every possible property state so we don't typically test every state in our beaker tests (unless there's a good reason to do so). We normally just do a default test (for properties that support default) and a non-default test (of each property). Additional tests are only done if we have incompatible properties that have to be tested separately.